charts/px-license-server/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+description: A Helm chart for install and configure PX-License-Server on Kubernetes and Openshift
+home: https://portworx.com/
+icon: https://raw.githubusercontent.com/portworx/helm/master/doc/media/px-logo.png
+keywords:
+- px-central
+- px-license-server
+name: px-license-server
+sources:
+- https://github.com/portworx/helm/tree/master/charts/px-license-server
+version: 1.0.0
```

Comment:
Should we keep this version inline with px-backup chart version?

Reply:
We have released 2 versions of px-backup chart 1.0.0 and 1.0.1
What I am thinking is to manage different releases of each chart but again if you want we can inline with a px-backup chart version.

Reply:
Yes